calico/about/about-calico.mdx

```diff
@@ -0,0 +1,68 @@
+
+- [AKS](../getting-started/kubernetes/managed-public-cloud/aks.mdx)
+- [Azure](../getting-started/kubernetes/self-managed-public-cloud/azure.mdx)
+- [Digital Ocean](../getting-started/kubernetes/self-managed-public-cloud/do.mdx)
+- [EKS](../getting-started/kubernetes/managed-public-cloud/eks.mdx)
+- [Flannel/Canal](../getting-started/kubernetes/flannel/install-for-flannel.mdx)
+- [GCE](../getting-started/kubernetes/self-managed-public-cloud/gce.mdx)
+- [GKE](../getting-started/kubernetes/managed-public-cloud/gke.mdx)
+- [IKS](../getting-started/kubernetes/managed-public-cloud/iks.mdx)
+- [K3s](../getting-started/kubernetes/k3s/index.mdx)
+- [kOps on AWS](../getting-started/kubernetes/self-managed-public-cloud/aws.mdx)
+- [Kubernetes-kubeadm](../getting-started/kubernetes/self-managed-onprem/onpremises.mdx)
+- [MicroK8s](../getting-started/kubernetes/microk8s.mdx)
+- [Minikube](../getting-started/kubernetes/minikube.mdx)
+- [OpenShift](../getting-started/kubernetes/openshift/index.mdx)
+- [OpenStack](../getting-started/openstack/index.mdx)
+- [RKE](../getting-started/kubernetes/rancher.mdx)
+- [Calico for Windows](../getting-started/kubernetes/windows-calico/index.mdx)
+
+## Installation options
+
+You can install {{prodname}} using Kubernetes operators, manifests (YAMLs), or Helm charts (Helm 3). 
+
+There are several quickstart options to start working with {{prodname}} in under 15 minutes:
+
+- [Quickstart for Calico on Kubernetes](../getting-started/kubernetes/quickstart.mdx)
+- [Quickstart for Calico on K3s](../getting-started/kubernetes/k3s/quickstart.mdx)
+- [Quickstart for Calico on minikube](../getting-started/kubernetes/minikube.mdx)
+- [Quickstart for Calico on MicroK8s](../getting-started/kubernetes/microk8s.mdx)
+
+## Feature summary
+
+The following table summarizes the main {{prodname}} features. To see a detailed feature comparison, see [Product comparison](../about/product-comparison.mdx#product-comparison-by-feature). 
+
+| Feature                      | Description                                                  |
+| ---------------------------- | ------------------------------------------------------------ |
+| Dataplanes                   | eBPF, standard Linux iptables, and Windows HNS.              |
+| Install                      | Install on Kubernetes clusters and non-cluster hosts.        |
+| Upgrade                      | Seamless upgrade to **Calico Cloud** and **Calico Enterprise**. |
+| Networking                   | - Scalable pod networking using BPG or overlay networking<br />- Customizable advanced IP address management |
+| Security                     | - Network policy enforcement for workload and host endpoints<br />- Data-in-transit encryption using WireGuard |
+| Monitoring Calico components | Monitor Calico component metrics using Prometheus.           |
+| User interfaces              | CLIs: `kubectl` and `calicoctl`                              |
```

Comment:
I'm not sure "User interfaces" is really a feature. If anything, I'd merge it with the APIs section.

e.g.,

"Fully configurable via Kubernetes-native REST APIs using kubectl, calicoctl, or the provided client library."

Reply:
Fixed.